doc/ref_model/README.rst

```diff
@@ -1,5 +1,6 @@
-Reference Model
-===============
+====================
+Reference Model (RM)
```

Comment:
For consistency with the RAs

```suggestion
Reference Model for cloud infrastructure (RM)
```

Reply:
Keep capital letters for Cloud and Infrastructure

Reply:
Fixed in [5ae8fba](https://github.com/cntt-n/CNTT/pull/3112/commits/5ae8fba5e37978ffb074d2bc4e884f2493eea2ef)